### PR TITLE
add unittest for pam auth

### PR DESCRIPTION
**Pull request by Yancey0623**
third task of #49

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,4 +26,5 @@ script:
   # one changing the Docker image, which must be merged before the
   # other one that changes the source code to be CI'ed with the
   # updated Docker image.
-  - docker run --rm -v $GOPATH:/go -w /go/src/github.com/sql-machine-learning/gohive sqlflow/gohive:dev bash test.sh
+  - docker run --rm -v $GOPATH:/go -w /go/src/github.com/sql-machine-learning/gohive yancey1989/gohive:dev bash test.sh
```
Comment (Yancey0623) [inline .travis.yml]: Will tag the personal repo to sqlflow if this PR has been approved.

**Review by tonyyang-svail — APPROVED** [review]
LGTM!

File: .travis.yml (hunk 1 above)
Comment (Yancey0623) [reply .travis.yml]: done.

Comment (Yancey1989) [reply .travis.yml]: done.

**Review by tonyyang-svail — APPROVED** [review]

**Review by typhoonzero — APPROVED** [review]

**Pull request merged**